### PR TITLE
[ROCm] Update ROCm and MigraphX CI to ROCm5.4

### PR DESCRIPTION
**Pull request by PeixuanZuo**
Update ROCm and MigraphX CI to ROCm5.4
Run ortmodule_test with ROCm5.4 and all passed(https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=824742&view=logs&j=8292f886-7946-5da9-7977-04484c342eda&t=5de68eaa-cbdc-5be5-13d0-bb946f4ddb2d).

### CODE REVIEW

File: tools/ci_build/github/azure-pipelines/linux-migraphx-ci-pipeline.yml [hunk 1]
```diff
@@ -8,14 +8,14 @@ jobs:
   pool: 'AMD-GPU'
   timeoutInMinutes: 180
 
-  # gid of video and render group on gcr-openpai-35 and -36
+  # gid of video and render group on gcramdrr1-mi100-085 and -86
```
Comment (ytaous) [inline tools/ci_build/github/azure-pipelines/linux-migraphx-ci-pipeline.yml:11]: >gid [](http://example.com/codeflow?start=4&length=3)

nit - get

File: tools/ci_build/github/azure-pipelines/orttraining-pai-ci-pipeline.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ jobs:
   pool: 'AMD-GPU'
   timeoutInMinutes: 150
 
-  # gid of video and render group on gcr-openpai-35 and -36
+  # gid of video and render group on gcramdrr1-mi100-085 and -86
```
Comment (ytaous) [inline tools/ci_build/github/azure-pipelines/orttraining-pai-ci-pipeline.yml:11]: >gid [](http://example.com/codeflow?start=4&length=3)

nit - get

Comment (ytaous) [inline tools/ci_build/github/azure-pipelines/orttraining-pai-ci-pipeline.yml:11]: >gid [](http://example.com/codeflow?start=4&length=3)

qq - has this part tested ?

**Comment by ytaous** [comment]
>   # For every upgrade, we should enable it to test it once, and disable it.

qq - has this been tested?

---
Refers to: tools/ci_build/github/azure-pipelines/orttraining-pai-ci-pipeline.yml:363 in b3bf703. [](commit_id = b3bf7036d6dd669ba5030cfc1a33ebc2d9a6b1f5, deletion_comment = False)

**Comment by PeixuanZuo** [comment]
> > # For every upgrade, we should enable it to test it once, and disable it.
> 
> qq - has this been tested?
> 
> Refers to: tools/ci_build/github/azure-pipelines/orttraining-pai-ci-pipeline.yml:363 in [b3bf703](https://github.com/microsoft/onnxruntime/commit/b3bf7036d6dd669ba5030cfc1a33ebc2d9a6b1f5). [](commit_id = [b3bf703](https://github.com/microsoft/onnxruntime/commit/b3bf7036d6dd669ba5030cfc1a33ebc2d9a6b1f5), deletion_comment = False)

Yes, It has been tested: https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=824742&view=logs&j=8292f886-7946-5da9-7977-04484c342eda&t=5de68eaa-cbdc-5be5-13d0-bb946f4ddb2d

File: tools/ci_build/github/azure-pipelines/linux-migraphx-ci-pipeline.yml (hunk 1 above)
Comment (PeixuanZuo) [reply tools/ci_build/github/azure-pipelines/linux-migraphx-ci-pipeline.yml:11]: The gid means group id, This comment means that the vaule of the variable video is it's group id.

**Review by snnn — APPROVED** [review]

**Pull request merged**